### ISSUE TITLE
Add medals for SS

### ISSUE THREAD

**Issue by abraker95** [ppy/osu-web#9690]
I believe this one has been requested for a while now here and there but there was never a formal issue for it

Applies for all gamemodes

**Comment by Felixland** [comment]
I second this. It allows medal hunters to have more medals to complete for thier collection. As well as show off the skills of players who can SS ridiculously hard maps.

**Comment by EXtremeExploit** [comment]
keep the bad ideas out of here we already have skylord ok thx

**Comment by minusQuantumNeko** [comment]
I totally agree with @EXtremeExploit

**Comment by chromb** [comment]
I second the idea of SS medals, but would much prefer SS count medals over Star Rating medals, might be interesting to get some community feedback on this one.

**Comment by minusQuantumNeko** [comment]
> I second the idea of SS medals, but would much prefer SS count medals over Star Rating medals, might be interesting to get some community feedback on this one.

On this one I can agree with you, SS count would be actually fun, unlike "10* SS medal", which obviously is a nonsense

**Comment by WitherFlower** [comment]
Indeed, SS players tend to focus on getting the highest amount of SS ranks rather than trying to SS high star rating maps

**Comment by TomyDoesThings** [comment]
We already have enough medals that require an SS. An SS tier to skill medals is completely unreasonable. We talk about passes and FCs. We don't really talk about our first SS on a star rating. And there's a reason why SS medals weren't introduced to begin with.
9* and 10* FC were already pushing it pretty damn far. SS tier would be very very foolish to do. 1 Skylord is more than enough.

**Comment by Corne2Plum3** [comment]
Yes but please don't go above 7 stars please

**Comment by minusQuantumNeko** [comment]
> Yes but please don't go above 7 stars please

Not going above 7* would make it look like a mess, so it's either all (up to 10 star) or none
